jwst/master_background/master_background_step.py

```diff
@@ -154,8 +154,8 @@ def process(self, input):
 
                 # Save the computed background if requested by user
                 if self.save_background:
-                    self.save_model(master_background, suffix='masterbg1d', asn_id=asn_id)
-                    self.save_model(background_2d_collection, suffix='masterbg2d', asn_id=asn_id)
+                    self.save_model(master_background, suffix='masterbg1d', force=True, asn_id=asn_id)
```

Comment:
Just curious as to why the "force" option is needed here? Does that cause it to overwrite an existing file that might already be there?